submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -58,9 +60,13 @@ open class SubmissionSubmitter(
         method: SubmissionMethod
     ): ExtSubmission {
         try {
-            return processSubmission(submission, user, source, method)
-        } catch (e: RuntimeException) {
-            throw InvalidSubmissionException("Submission validation errors", listOf(e))
+            val submitted = processSubmission(submission, user, source, method)
+            successfulSubmission.onNext(SuccessfulSubmission(
```

Comment:
here the submission has not been submitted yet. I think this should be after the persistence layer process getscompleted.